src/http/request_handler.rs

```diff
@@ -37,9 +37,9 @@ fn update_cache_control_header(
   req_ctx: Arc<RequestContext>,
 ) -> GraphQLResponse {
   if server_ctx.blueprint.server.enable_cache_control_header {
-    if let Some(ttl) = req_ctx.get_min_max_age() {
-      return response.set_cache_control(ttl as i32);
-    }
+    let ttl = req_ctx.get_min_max_age().unwrap_or(0);
+    let cache_private_flag = req_ctx.is_cache_private().unwrap_or(false);
+    return response.set_cache_control(ttl as i32, cache_private_flag);
```

Comment:
```suggestion
    let ttl: i32 = req_ctx.get_min_max_age().unwrap_or(0);
    let cache_private_flag = req_ctx.is_cache_private().unwrap_or(false);
    return response.set_cache_control(ttl, cache_private_flag);
```

Reply:
This threw an error while running tests. So, I updated everything to just directly use `i32` since that's the type we use at the end.